.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!-- Thank you for submitting a pull request to our repo. -->
+
+<!-- If this is your first PR in this repo, please read our [Contributing Guidelines (https://github.com/SteeltoeOSS/.github/blob/master/CONTRIBUTING.md) and remember to sign the [Contributor License Agreement](https://github.com/SteeltoeOSS/.github/blob/main/contributing-docs/contributing-license.md). Our bot will notify you shortly after the PR has been created. -->
+
+## Summary
+
+{Summary of the changes, less than 80 chars}
+
+## Description
```

Comment:
```suggestion
## Description <!-- optional, no need to duplicate the summary -->
```

Reply:
How about we drop the Summary/Description and replace it with just Description? The "less than 80 chars" feels a bit patronizing anyway.

Reply:
Yes, that works